runtime/doc/nvim_python.txt

```diff
@@ -24,10 +24,15 @@ simple tutorial:
 
 - Make sure Python 2.6 or 2.7 is available in your `$PATH`
 - Install the `neovim` Python package:
-  >
-      $ pip install neovim
+>
+    $ pip install neovim
 <
 Most Python plugins created for Vim 7.3 should work after these steps.
 
+To point Nvim to a specific Python interpreter, set `g:python_host_prog`:
+>
+    let g:python_host_prog='/path/to/python'
+<
+
```

Comment:
Or wait, there's always a blank line at the end of the docs.. will this result in two blank lines because of the `<` on its own line?